magento/magento2
# Configurabel products special price bug 10303

### Description (*)
vendor/magento/module-swatches/view/frontend/web/js/swatch-renderer.js
-            selectorProduct: '.product-info-main',
+            selectorProduct: '.--product-info-main',

### Fixed Issues (if relevant)
1. magento/magento2#10303: Configurabel products special price bug
2. ...

### Manual testing scenarios (*)

1. Create configurable product with simple products
2. Add special Prices to Simple products associated with the configurable product
3. Clear Cache
4. Go to Product detail page
5. Change the configurable option drop-down(Ex:size) and check Price
6. Special Price of simple product should be displayed

### Contribution checklist (*)
